public/files/69-probe-rs.rules

```diff
@@ -0,0 +1,146 @@
+# Copy this file to /etc/udev/rules.d/
+# If rules fail to reload automatically, you can refresh udev rules
+# with the command "udevadm control --reload"
+
+# This rules are based on the udev rules from the OpenOCD project, with unsupported probes removed.
+# See http://openocd.org/ for more details.
+#
+# This file is available under the GNU General Public License v2.0
+
+ACTION!="add|change", GOTO="probe_rs_rules_end"
+
+SUBSYSTEM=="gpio", MODE="0660", GROUP="plugdev", TAG+="uaccess"
+
+SUBSYSTEM!="usb|tty|hidraw", GOTO="probe_rs_rules_end"
+
+# Please keep this list sorted by VID:PID
+
+# STMicroelectronics ST-LINK V1
+ATTRS {
+    idVendo
+}
+==#$#%#$#placeholder03434#$#%#$#, ATTRS{idProduct}==#$#%#$#placeholder13434#$#%#$#, MODE=#$#%#$#placeholder23434#$#%#$#, GROUP=#$#%#$#placeholder33434#$#%#$#, TAG+=#$#%#$#placeholder43434#$#%#$#
```

Comment:
Are those placeholders intended?

Reply:
Nope, not sure what happened there. Fixed here: https://github.com/probe-rs/webpage/pull/173